.appveyor.yml

```diff
@@ -7,7 +7,7 @@ matrix:
 
 install:
   - ps: Install-Product node $env:nodejs_version
-  - yarn install
+  - yarn --frozen-lockfile
```

Comment:
I believe `yarn` is an alias for `yarn install` according to the [official docs](https://yarnpkg.com/lang/en/docs/cli/install/)

> Running yarn with no command will run yarn install, passing through any provided flags.